docs/paper/dev/api/plugin-messaging.md

```diff
@@ -194,3 +195,29 @@ This is not a recommended way to ban players due to the fact that there may not
 but it is an example of how this can be used.
 
 :::
+
+#### MessageRaw
+
+The `MessageRaw` message type is used to send a raw chat component to a player. This is useful for sending clickable messages to players,
+or messages with hover events.
+
+```java
+public class MyPlugin extends JavaPlugin {
+    
+    @Override
+    public void onEnable() {
+        this.getServer().getMessenger().registerOutgoingPluginChannel(this, "BungeeCord");
+        
+        Player player = ...;
+        ByteArrayDataOutput out = ByteStreams.newDataOutput();
+        out.writeUTF("MessageRaw");
+        out.writeUTF("ALL");
```

Comment:
The text above the snippet make it sound like this is only going to one player, when this is being broadcasted to all players on the proxy. This should be clarified to mention that this should be the name of the target player or ALL.